.gitignore

```diff
@@ -73,3 +73,4 @@ terraform.rc
 # Vulnerability management
 db/
 fanal/
+java-db/
```

Comment:
not sure if it's smth specific to my machine or not, but I had this untracked dir after running cloudbeat locally with vulnerabilities yaml